packages/haul-babel-preset-react-native/src/index.ts

```diff
@@ -30,7 +30,16 @@ export default function getHaulBabelPreset() {
         plugins: [require('@babel/plugin-transform-flow-strip-types')],
       },
       {
-        plugins: defaultPlugins,
+        plugins: defaultPlugins.concat(
+          process.env.HAUL_PLATFORM
```

Comment:
Can you make `platform` a preset config? This is hacky

Reply:
It is but I cannot pass this value (`platform`) in a normal way as a preset option.

Reply:
wtf Babel

Reply:
I'm not sure it's babel fault, `platform` is like a dynamic value, we cannot hardcode it in the Babel config nor we can pass additional metadata to Babel plugins through `babel-loader`.

Reply:
can you document it somehow somewhere?